### ISSUE TITLE
custom min/max FontSize

### ISSUE THREAD

**Issue by azazadev** [mehdizarepour/instagram-text-editor#7]
it's possible to add new params for fontSize min/max value in TextEditor ? as of now this is hardcoded 

```
@override
  Widget build(BuildContext context) {
    return Consumer<TextStyleModel>(
      builder: (context, model, child) => RotatedBox(
        quarterTurns: 3,
        child: Slider(
          value: model.textStyle?.fontSize ?? 1,
          min: 0,
          max: 100,
          divisions: 100,
          activeColor: Colors.white,
          inactiveColor: Colors.white,
          onChanged: (double value) => model.editFontSize(value),
        ),
      ),
    );
  }
```

**Comment by mehdizarepour** [comment]
Hi @azazadev, Thank you for opening this issue.
Yes, this is hardcoded but it's easy to add these params. I'll do it as soon as I can, also you are welcome to contribute.

**Comment by mehdizarepour** [comment]
@azazadev This feature is available on `v0.4.1`.

https://github.com/mehdizarepour/instagram-text-editor/blob/98969f0357043bfb5b32d708a569ed39a92c5a4f/example/lib/main.dart#L80

I'm closing this issue, feel free to reopen it if you have any problem.
thanks.

**Comment by azazadev** [comment]
awesome ! work as expected
